src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.CountryDao;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
 
 public class CountryServiceImpl implements CountryService {
+
+    private CountryDao countryDao;
+
+    public CountryServiceImpl(CountryDao countryDao) {
+        this.countryDao = countryDao;
+    }
+
+    public CountryServiceImpl() {
+    }
```

Comment:
The no-argument constructor `CountryServiceImpl()` is unnecessary and potentially problematic because the `CountryDao` is required for the service to function. Consider removing this constructor to enforce proper initialization.

Reply:
ok, done